client/signup/steps/site-style/index.jsx

```diff
@@ -124,15 +139,21 @@ export class SiteStyleStep extends Component {
 					fallbackSubHeaderText={ subHeaderText }
 					signupProgress={ signupProgress }
 					stepContent={ this.renderContent() }
+					showSiteMockups={ showSiteMockups }
 				/>
 			</div>
 		);
 	}
 }
 
 const mapDispatchToProps = ( dispatch, ownProps ) => ( {
-	submitSiteStyle: ( siteStyle, themeSlugWithRepo ) => {
+	submitSiteStyle: ( siteStyle, themeSlugWithRepo, styleLabel ) => {
 		const { flowName, stepName, goToNextStep } = ownProps;
+		dispatch(
+			recordTracksEvent( 'calypso_signup_actions_submit_site_style', {
+				site_style: styleLabel,
```

Comment:
Any specific reason that the label is better than the `siteStyle` here?
As long as the site label won't be localized in the future and is as stable as the style id slug, I'm fine with keeping it this way.

Reply:
Thanks for the review @southp !!!

> Any specific reason that the label is better than the siteStyle here?

Good question.  though it might be more descriptive as `siteStyle` values, such as 'modern' or 'default' are quite generic and might be used elsewhere. I might be wrong. 

The site label is the theme descriptor, so in the future it could be the name of the theme or other "product-like" label, which we don't translate. For example: https://it.wordpress.org/themes/

Having said that, I should probably make a comment to that effect. We're only using it for 4 (untranslated) strings atm.